Mathlib/RingTheory/Ideal/Maps.lean

```diff
@@ -385,6 +398,54 @@ theorem mem_map_of_equiv {E : Type*} [EquivLike E R S] [RingEquivClass E R S] (e
   · rintro ⟨x, hx, rfl⟩
     exact mem_map_of_mem e hx
 
+section Bijective
+
+variable (hf : Function.Bijective f) {I : Ideal R} {K : Ideal S}
+include hf
+
+/-- Special case of the correspondence theorem for isomorphic rings -/
+def relIsoOfBijective : Ideal S ≃o Ideal R where
+  toFun := comap f
+  invFun := map f
+  left_inv := map_comap_of_surjective _ hf.2
+  right_inv J := le_antisymm (fun _ h ↦
+    have ⟨y, hy, eq⟩ := (mem_map_iff_of_surjective _ hf.2).mp h; hf.1 eq ▸ hy) le_comap_map
```

Comment:
I think this would be more readable as
```suggestion
  right_inv J :=
    le_antisymm
      (fun _ h ↦ have ⟨y, hy, eq⟩ := (mem_map_iff_of_surjective _ hf.2).mp h; hf.1 eq ▸ hy)
      le_comap_map
```